src/test/resources/data/typicalDataBundle.json

```diff
@@ -578,6 +647,15 @@
       "team": "Team 2.1",
       "section": "None"
     },
+    "student1InCourse3": {
```

Comment:
Since this student has a googleId, that means this student details should be present in `account` array also